packages/design-system/src/components/Form/Primitives/Input/Input.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {
+	forwardRef,
+	Ref,
+	InputHTMLAttributes,
+	useState,
+	FocusEvent,
+	useRef,
+	useImperativeHandle,
+} from 'react';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import InputWrapper, { AffixesProps } from '../InputWrapper/InputWrapper';
+import Tooltip from '../../../Tooltip';
+import Clickable from '../../../Clickable';
+import { Icon } from '../../../Icon/Icon';
+
+type InputProps = Omit<InputHTMLAttributes<any>, 'prefix' | 'suffix'> & AffixesProps;
+
+import styles from './Input.module.scss';
+
+const Input = forwardRef((props: InputProps, ref: Ref<HTMLInputElement | null>) => {
+	const { className, prefix, suffix, readOnly, disabled, type, onBlur, ...rest } = props;
+
+	// Password type management
+	const [isClear, setClear] = useState<boolean>(type !== 'password');
+	const inputType = type === 'password' ? (isClear ? 'text' : type) : type;
+	const inputRef = useRef<HTMLInputElement | null>(null);
+	const { t } = useTranslation();
+	const showMsg = t('FORM_PASSWORD_SHOW', { defaultValue: 'Show password' });
+	const hideMsg = t('FORM_PASSWORD_HIDE', { defaultValue: 'Hide password' });
+
+	useImperativeHandle(ref, () => inputRef.current);
+
+	function handleBlur(event: FocusEvent<any, HTMLInputElement>) {
+		setClear(false);
+		if (onBlur) {
+			onBlur(event);
+		}
+	}
+
+	return (
+		<InputWrapper prefix={prefix} suffix={suffix} disabled={!!disabled} readOnly={!!readOnly}>
+			<>
```

Comment:
you don't need this Fragment

Reply:
I do, InputWrapper takes only one child.